perfkitbenchmarker/linux_packages/cassandra.py

```diff
@@ -90,6 +94,27 @@ def AptInstall(vm):
   _Install(vm)
 
 
+def JujuInstall(vm):
+  if vm.isController is True:
+    if vm.vm_group_specs and 'workers' in vm.vm_group_specs:
+      vm.JujuDeploy('cs:trusty/cassandra',
+                    vm.vm_group_specs['workers'].vm_count,
+                    'workers')
```

Comment:
It sounds like the `cassandra_stress` tool needs to be its own package if that's the case. The package itself should not need knowledge of the benchmark. This [line](https://github.com/GoogleCloudPlatform/PerfKitBenchmarker/blob/master/perfkitbenchmarker/linux_benchmarks/cassandra_stress_benchmark.py#L320) should change to
```
vm_util.RunThreaded(lambda vm: vm.Install('cassandra'), benchmark_spec.vm_groups['workers'])
vm_util.RunThreaded(lambda vm: vm.Install('cassandra_stress'), benchmark_spec.vm_groups['client'])
```

Your `JujuInstall(controller, group_name)` function should only install charms on one group. That way if I want to reuse the `cassandra` package in another benchmark I can install the charm on a single group without having `cassandra_stress` getting installed on any of the other vms